src/System.Private.CoreLib/Resources/Strings.resx

```diff
@@ -2224,12 +2220,6 @@
   <data name="EventSource_VarArgsParameterMismatch" xml:space="preserve">
     <value>The parameters to the Event method do not match the parameters to the WriteEvent method. This may cause the event to be displayed incorrectly.</value>
   </data>
-  <data name="Exception_EndOfInnerExceptionStack" xml:space="preserve">
```

Comment:
It is not unusual for folks to have parser of the `Exception.ToString` message - to reformat it or to identify root cause. Here is an example found by a quick github search:

https://github.com/pieceofsummer/Hangfire.StackTrace/blob/45ced0e988bee23f86d71c2fa60997b9210f677a/src/Hangfire.StackTrace/FailedStateRenderer.cs#L146

This change is likely going to break these ... that may be ok, but we do not have much time to react to feedback in 3.0.

Reply:
Cant we just enable that through configuration. In that way, people can start preparing for it, like it was done with tiered JIT.

Reply:
We have used AppContext switches less than a dozen times in .NET Core so far. If it is a temporary thing we take out after 3.0 then I do not object. I can use a switch to hide most of this just for a few weeks. Of course that means that few if anyone "benefits" from this in 3.0.

Another option is that I keep the changes to put messages on their own lines, and to avoid duplicating the first inner exception in the AggregateException, but defer the change to indent and to remove the two dividers. 

thoughts?

Reply:
I moved part into https://github.com/dotnet/coreclr/pull/25185 for 3.0, and will pick this up again when master is open for .NET 5.